src/docs/sphinx/advancedExamples/validationStudies/mccWellbore/Example.rst

```diff
@@ -0,0 +1,235 @@
+.. _ExampleMCCWellbore:
+
+
+####################################################
+Modified Cam-Clay Model for Wellbore Problems
+####################################################
+
+
+**Context**
+
+In this benchmark example, the Modified Cam-Clay model (see :ref:`ModifiedCamClay`) is applied to solve for elastoplastic deformation within the vicinity of a vertical wellbore. For the presented example, an analytical solution is employed to verify the accuracy of the numerical results. The resulting model can be used as a base for more complex analysis (e.g., wellbore drilling, fluid injection and storage scenarios). 
+
+
+**Input file**
+
+Everything required is contained within two GEOSX input files located at:
+
+.. code-block:: console
+
+  src/coreComponents/physicsSolvers/solidMechanics/benchmarks/PlasticWell_Base.xml
+  src/coreComponents/physicsSolvers/solidMechanics/benchmarks/ModifiedCamClay_Verification.xml
+
+
+------------------------------------------------------------------
+Description of the case
+------------------------------------------------------------------
+
+We simulate a drained wellbore problem subjected to isotropic horizontal stress (:math:`\sigma_h`) and vertical stress (:math:`\sigma_v`), as shown below. By increasing the wellbore supporting pressure (:math:`P_w`), the wellbore expands, and the formation rock experiences elastoplastic deformation. A plastic zone develops in the near wellbore region.
+
+
+.. _problemSketchFig:
+.. figure:: MCC_wellSketch.png
+   :align: center
+   :width: 500
+   :figclass: align-center
+
+   Sketch of the wellbore problem 
+
+
+To simulate this phenomenon, the Modified Cam-Clay model is used in this example. Displacement and stress fields around the wellbore are numerically calculated. These numerical predictions are then compared with the corresponding analytical solutions `(Chen and Abousleiman, 2013)  <https://www.icevirtuallibrary.com/doi/10.1680/geot.11.P.088>`__ from the literature. 
+
+
+
+In this example, we focus our attention on the ``Mesh`` tags,
+the ``Constitutive`` tags, and the ``FieldSpecifications`` tags.
+
+------------------------------------------------------------------
+Mesh
+------------------------------------------------------------------
+
+Following figure shows the generated mesh that is used for solving this wellbore problem.
+
+
+.. _problemSketchFig:
+.. figure:: MCC_wellMesh.png
+   :align: center
+   :width: 500
+   :figclass: align-center
+
+   Generated mesh for the wellbore problem
+
+
+Let us take a closer look at the geometry of this wellbore problem.
+We use the internal mesh generator ``InternalWellbore`` to create a rock domain
+(:math:`10\, m \, \times 5 \,  m \, \times 2 \, m`), with a wellbore of
```

Comment:
Please verify this

Reply:
This is the right size of the simulated domain (only half of the entire one).